README.md

```diff
@@ -60,7 +60,18 @@ ENV["MYSQL_DATABASE"] = "test_db"
 
 ## Development
 
-After cloning run `crystal deps` or `crystal deps update`.
+After cloning makes scripts executable:
+
+```
+chmod a+x bin/*
+chmod a+x script/setup-test-db.sh
```

Comment:
What is the reason for these chmods? 

From my knowledge git stores executable flag as part of commit. So if somebody checks out the file, that was already made executable before checking it in, it should just work. At least, for Linux and MacOS.

Is there some context that I am missing? Some shell, git version, git config or OS that I am not aware of working differently in the regard of executable flags?